crates/holochain/src/core/ribosome/host_fn/call_remote.rs

```diff
@@ -15,8 +15,11 @@ pub fn call_remote(
     match HostFnAccess::from(&call_context.host_context()) {
         HostFnAccess {
             write_network: Permission::Allow,
+            agent_info: Permission::Allow,
             ..
         } => {
+            let from_agent = super::agent_info::agent_info(_ribosome, call_context.clone(), ())?
+                .agent_latest_pubkey;
```

Comment:
just curiosity, why is it "latest", does the pubkey iterate somehow in this context?

Reply:
@neonphog yeah there's an "initial" as well